cdap-data-fabric/src/test/java/co/cask/cdap/data2/metadata/dataset/MetadataKeyTest.java

```diff
@@ -59,6 +60,44 @@ public void testGetMetadataKey() {
     Assert.assertEquals("key1", MetadataKey.extractMetadataKey(mdsIndexKey.getKey()));
   }
 
+  @Test
```

Comment:
it would be nice to add a comment here, what this test is for. If I understand correctly, it tests that the meta data store ignores the version part of apps, programs, and schedules?

Reply:
Yes. Correct. Will add a comment explaining it.

Reply:
Done.